### DISCUSSION TITLE
Will chakra-ui and ark-ui be related in the future?

### DISCUSSION THREAD

**Discussion by ShumRain** [chakra-ui/ark#698] [Q&A] [answered]
Will chakra-ui and ark-ui be related in the future?

**Comment by cschroeter** [comment, marked as answer]
Ark will serve as the core of Chakra UI moving forward. We'll begin by adding new components like pagination, datepicker etc. and subsequently, replace existing Chakra UI components with Ark based components.

As for the relationship between Chakra UI and Ark UI in the future, it's safe to say that they will be closely related, with Ark being the underlying foundation for Chakra UI.
